Backend/templates/index.html

```diff
@@ -0,0 +1,32 @@
+<html>
+<head>
+  <link rel = stylesheet type = text/css href = "{{url_for('static', filename = 'style.css')}}">
+</head>
+  <body>
+    <h2>Welcome to Your Lean Fridge!</h2>
+    <div id = 'page-wrap'>
+    <p>Got a new item? Add it <a href = '{{url_for('newItem', stock = stock)}}'>here</a>
+      {% with messages = get_flashed_messages() %}
+      {% if messages %}
+      <!--To display all the flashed messages in an unordered list -->
+      <ul>
+        {% for message in messages %}
+        <li><strong>{{message}}</strong></li>
+        {% endfor %}
+      </ul>
+      {% endif %}
+      {% endwith %}
+
+
+    {% for item in stock %}
```

Comment:
the indentation here looks weird. block 22 to 28 should be indented more than line 21, isn't it ?